### ISSUE TITLE
Missing humble-devel branch

### ISSUE THREAD

**Issue by fvlvivs** [locusrobotics/fuse#339]
Dear all,

We are going to introduce IMU on our 2D mobile robot. An online search showed that **fuse** is going to be considered the future for sensor fusion in the ROS framework. Our software is based on the **humble** distro, and we noticed that **fuse** misses that. Is there any intention to have this package on **humble** too?

**Comment by johnny555** [comment]
I think it won't work because fuse seems to depend upon API changes introduced in Iron. Specifically: 

https://docs.ros.org/en/humble/Releases/Release-Iron-Irwini.html#addition-of-a-new-paradigm-for-handling-node-and-lifecyclenode 

I'm pretty sure this is why I get errors like (when compiling on humble): 

```
src/fuse/fuse_core/include/fuse_core/node_interfaces/node_interfaces.hpp:20:10: fatal error: rclcpp/node_interfaces/node_interfaces.hpp: No such file or directory
   20 | #include <rclcpp/node_interfaces/node_interfaces.hpp>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

I think we would need to back port the node/lifecycle node interfaces to humble if we want to use Fuse, but I could be wrong. 

There was also a backport request https://github.com/ros2/rclcpp/issues/2309 however it looks like it never happened.

**Comment by giafranchini** [comment]
Hi, last year me and a collegue of mine backported some features of rclcpp in order to compile fuse in Humble, maybe can be useful to you too. If you are interested I did a fork back then https://github.com/giafranchini/rclcpp/tree/humble

**Comment by sukhrajklair** [comment]
@giafranchini I tried to build with your fork of rclcpp but the fuse_publishers package failed to build as it relies on tf2_ros. Have you managed to build fuse for humble?

**Comment by giafranchini** [comment]
Yes you're right, we also did that backport I just didn't remember since then we switched to iron. Here it is: https://github.com/giafranchini/geometry2/tree/humble
